Web/MVCAssignment/WebContent/View/Register.jsp

```diff
@@ -0,0 +1,84 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+	pageEncoding="ISO-8859-1"%>
+<%@ page import="com.Model.*"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Register</title>
+
+<script type="text/javascript" src="../MVCAssignment/register.js"></script>
+</head>
+<body>
+	<header>
+	<h1>Edit/Register Employee Details</h1>
+	</header>
+	<ul>
+		<li><a
+			href="http://localhost:8080/MVCAssignment/View/LandingPage.jsp">Landing
```

Comment:
ACTION: How can we run this application on remote or different port, as you have defined statically  http://localhost:8080. Please modify all path accordingly.